### PR TITLE
Using Obsidian Callouts for Highlights and Annotations.

### PR DESCRIPTION
**Pull request by android272**
I changed the code to add the ability to turn on and off Obsidian callouts for highlights and annotations. Not only that but I also gave users the ability to change which callout to use for each highlights and annotations. You can turn this off to go back to the normal markdown block quotes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,12 @@ tags:
 ![](./README_assets/step3.png)
 ![](./README_assets/step4.png)
 
+## Obsidian Callouts
+You can turn off callouts to go back to normal markdown block quotes as well as change which callout to use for highlights and annotations.
```
Comment (OGKevin) [inline README.md:54]: Could you please link to the following doc: https://help.obsidian.md/How+to/Use+callouts

File: src/settings/Settings.ts [hunk 2]
```diff
@@ -88,4 +97,45 @@ export class KoboHighlightsImporterSettingsTab extends PluginSettingTab {
                     })
             })
     }
+
+    add_enable_callouts(): void {
+        new Setting(this.containerEl)
+            .setName("Use Callouts")
+            .setDesc(`If the exported higlights should use callouts ex: '> [!quote]' & '> [!note]'`)
```
Comment (OGKevin) [inline src/settings/Settings.ts:104]: Is it possible to link to the docs here as well? https://help.obsidian.md/How+to/Use+callouts

**Review by OGKevin — CHANGES_REQUESTED** [review]
Thanks for this PR!

Do you mind adding a few test cases like the following: https://github.com/OGKevin/obsidian-kobo-highlights-import/blob/e1f46a30d73322f27f01b78a50385046855cb955/src/database/Highlight.test.ts#L34-L60

I know you can't run the test, as they are run with my Database, but if you could just write the test cases would be great. I can then run the test and make the adjustments needed to make it pass, if some fail.

**Comment by OGKevin** [comment]
Alright, @android272 if you could merge: https://github.com/android272/obsidian-kobo-highlights-import/pull/1 then this one can be merged as well 👍🏾

**Comment by android272** [comment]
Yeah, I just have a few more edits to do before I merge.

**Comment by android272** [comment]
Is there anything that you need me to do for this?

**Comment by OGKevin** [comment]
Hey Andrew, I’ve been on a long vacation without 💻. I’ll be back next week and will have a look then! On 3 Jan 2023, at 04:19, Andrew Pullins ***@***.***> wrote:﻿
Is there anything that you need me to do for this?

**Pull request merged**